Formula/libadwaita-1.rb

```diff
@@ -0,0 +1,50 @@
+class Libadwaita1 < Formula
+  desc "Building blocks for modern adaptive GNOME applications"
+  homepage "https://gnome.pages.gitlab.gnome.org/libadwaita/"
+  url "https://download.gnome.org/sources/libadwaita/1.0/libadwaita-1.0.1.tar.xz"
+  sha256 "bb49cf5a09d2e8bc144946c2c3272aecd611667fd027f3808b95d7101ed473d6"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    url :stable
+    regex(/libadwaita[._-](1\.([0-8]\d*?)?[02468](?:\.\d+)*?)\.t/i)
+  end
```

Comment:
Did you try to do `brew livecheck` without this? We already have a default strategy for GNOME which checks for stable versions.

Reply:
Thanks. I removed this from the updated commit. libadwaita will not even use the even/odd versioning scheme of GNOME (at least I think so). `brew livecheck` still works without it.